src/trusted-device/Security/TwoFactor/Trusted/TrustedDeviceToken.php

```diff
@@ -4,38 +4,38 @@
 
 namespace Scheb\TwoFactorBundle\Security\TwoFactor\Trusted;
 
-use Lcobucci\JWT\Token;
+use Lcobucci\JWT\Token\Plain;
 
 class TrustedDeviceToken
 {
     /**
-     * @var Token
+     * @var Plain
      */
     private $jwtToken;
 
-    public function __construct(Token $jwtToken)
+    public function __construct(Plain $jwtToken)
```

Comment:
Officially, this class is not internal, do you consider this a BC break? If you do, we can leave the typehint as-is (v4 has `Token` as interface) and throw an `InvalidArgumentException` if it's not a plain token (we require a plain token, as `claims()` is not in the `Token` interface)

Reply:
I consider these classes internal classes, they're an implementation detail rather than an extension point. So I'm fine with changing the method signature. Therefore let's keep `Plain` here.